poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.1 and should not be changed by hand.
```

Comment:
@aliberts There are a lot of changes here without changes to `pyproject.toml`. Should we expect this?

Reply:
There shouldn't be any changes to `poetry.lock`, it's probably due to having run something like `poetry lock`

I'd suggest to revert changes on the lock file.

I'm actually thinking of removing the `poetry.lock` file entirely from the repo.

Reply:
If I remember correctly, I updated the packages which could explain why the lock file changed. I reverted that change